Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -705,6 +705,14 @@ public void SetScaleLimits(float min, float max, bool relativeToInitialState = t
             }
         }
 
+        /// <summary>
+        /// Destroys and re-creates the rig around the bounding box
+        /// </summary>
+        public void UpdateRig()
```

Comment:
>UpdateRig [](start = 20, length = 9)

For simplicity, I would have just one method instead of having UpdateRig() being a public wrapper around CreateRig().